Tests/SwiftKueryTests/TestSchema.swift

```diff
@@ -71,6 +72,71 @@ class TestSchema: XCTestCase {
         let b = Column("b")
     }
 
+    func testMultipleForeignKeys() {
+        let connection = createConnection()
+
+        var t1 = Table1()
+        var t2 = Table2()
+
+        // This tests a one to one foreign key
+        var createStmt = createTable(t1.foreignKey(t1.a, references: t2.b), connection: connection)
```

Comment:
The names t1.a are not very helpful for understanding the tests but i realize that you have done this to be consistent with the other tests. Maybe we need another PR to rename all the tests so they can be read more easily

Reply:
I think the tests would benefit from a bit of an overhaul in future. I can raise an issue.